development/markdown-index/main.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"github.com/kyma-project/test-infra/development/markdown-index/bumper"
+	"github.com/sirupsen/logrus"
+	flag "github.com/spf13/pflag"
+	"log"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+	"strings"
+)
+
+const repositoryName = "test-infra"
```

Comment:
The repository name is also provided in the config, why it is hardcoded?

Reply:
fixed